apps/context.go

```diff
@@ -4,23 +4,57 @@ import (
 	"github.com/mattermost/mattermost-server/v5/model"
 )
 
+// Context is included in CallRequest and provides App with information about
+// Mattermost environment (configuration, authentication), and the context of
+// the user agent (current channel, etc.)
+//
+// To help reduce the need to go back to Mattermost REST API, ExpandedContext
+// can be included by adding a corresponding Expand attribute to the originating
+// Call.
 type Context struct {
-	AppID             AppID             `json:"app_id"`
-	Location          Location          `json:"location,omitempty"`
-	Subject           Subject           `json:"subject,omitempty"`
-	BotUserID         string            `json:"bot_user_id,omitempty"`
-	ActingUserID      string            `json:"acting_user_id,omitempty"`
-	UserID            string            `json:"user_id,omitempty"`
-	TeamID            string            `json:"team_id"`
-	ChannelID         string            `json:"channel_id,omitempty"`
-	PostID            string            `json:"post_id,omitempty"`
-	RootPostID        string            `json:"root_post_id,omitempty"`
-	Props             map[string]string `json:"props,omitempty"`
-	MattermostSiteURL string            `json:"mattermost_site_url"`
-	UserAgent         string            `json:"user_agent,omitempty"`
+	// AppID is used for handling CallRequest internally, and is always filled
```

Comment:
"AppID is used for handling CallRequest internally" - might help expand a bit on that.